packages/components/src/components/input-field/input.tsx

```diff
@@ -80,12 +80,12 @@ const Input = ({
     }, [current_focus, name]);
 
     const onBlurHandler = (e: React.FocusEvent<HTMLInputElement>) => {
-        setCurrentFocus(null);
+        setCurrentFocus?.(null);
         if (onBlur) {
             onBlur(e);
         }
```

Comment:
```suggestion
        if (setCurrentFocus) {
            setCurrentFocus(null);
        }
        if (onBlur) {
            onBlur(e);
        }
```

Nitpick: Is it better to be consistent this this sort of stuff.

Reply:
or onBlur.?(e) :)

Reply:
Yeah whatever you prefer. Just thinking being consistent makes it easier for me to read, maybe others are the same.

Reply:
@markwylde-deriv I hope you don't mind if I proceed with rewriting `onBlur.?(e)` as we use dot notation more often

Reply:
Thank you for suggestions